paasta_tools/vitesscluster_tools.py

```diff
@@ -48,8 +570,102 @@ def __init__(
             branch_dict=branch_dict,
         )
 
-    def get_instances(self, with_limit: bool = True) -> int:
-        return self.config_dict.get("replicas", 1)
+    def get_namespace(self) -> str:
+        return self.config_dict.get("namespace", KUBERNETES_NAMESPACE)
+
+    def get_images(self) -> Dict[str, str]:
+        vitess_images = self.config_dict.get(
+            "images", load_system_paasta_config().get_vitess_images()
+        )
+        return {
+            "vtctld": vitess_images["vtctld_image"],
+            "vtadmin": vitess_images["vtadmin_image"],
+            "vtgate": vitess_images["vtgate_image"],
+            "vttablet": vitess_images["vttablet_image"],
+        }
+
+    def get_globalLockserver(self) -> Dict[str, Dict[str, str]]:
+        zk_address = self.config_dict.get("zk_address")
+        return {
+            "external": {
+                "implementation": TOPO_IMPLEMENTATION,
+                "address": zk_address,
+                "rootPath": TOPO_GLOBAL_ROOT,
+            }
+        }
+
+    def get_cells(self) -> List[str]:
+        paasta_pool = self.get_pool()
+        paasta_cluster = self.config_dict.get("paasta_cluster", self.get_cluster())
+        cells = self.config_dict.get("cells")
+        region = self.config_dict.get("region")
+        vtgate_resources = self.config_dict.get("vtgate_resources")
+        return [  # type: ignore
```

Comment:
mypy is correct here - the return type for this function does not match what will actually be returned :)

Reply:
https://github.com/Yelp/paasta/pull/3889/commits/4bcdd469f1e5b47fd6ff85f18d3cb77bbcd44203